server/src/main/java/io/druid/server/listener/announcer/ListeningAnnouncerConfig.java

```diff
@@ -93,7 +93,7 @@ public String getAnnouncementPath(String listenerName)
   {
     return ZKPaths.makePath(
         getListenersPath(), Preconditions.checkNotNull(
```

Comment:
Each argument should be on a separate line

Reply:
done.